client/src/components/CardItem/CardItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './CardItem.module.scss'
+import {ReactComponent as Arrow} from "../../svg/hotDeals/arrow.svg";
+
+
+function CardItem() {
+    return (
+        <div className={styles.cardItem}>
+            <p className={styles.price}>$ 200.00</p>
+            <p className={styles.header}>KYIV CITY WALK</p>
```

Comment:
What about using `text-transform: uppercase` in styles instead of using upper case in html?